packages/dds/merge-tree/src/test/sortedSegmentSet.spec.ts

```diff
@@ -115,6 +115,7 @@ describe("SortedSegmentSet", () => {
 		assert.equal(set.size, client.getLength() * 2);
 		validateSet<LocalReferencePosition>(
 			client,
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
 			(set as any).trackedSet,
```

Comment:
Nit: Can we cast to a specifi type instead of `any`?

Reply:
No, I don't think we can because the type arguments of the two set types intentionally do not match up for this test. I added a comment to document it though.